README.md

```diff
@@ -153,6 +153,43 @@ TODO: complete me!
 
 ### `taskTemplate`
 
+* `name`: name of Nomad task group to run. It specifies value for
+  `TaskGroups/Name` in Nomad Job definition.
```

Comment:
I would link directly to the relevant Nomad job specification configuration in that case (which would be https://nomadproject.io/docs/job-specification/group/ , although there's no direct `name` attribute for a group.)